packages/ui/src/composables/useValidation.ts

```diff
@@ -50,8 +65,16 @@ export const useValidation = <V, P extends ExtractPropTypes<typeof useValidation
     computedErrorMessages.value = []
   }
 
+  const withoutValidation = (cb: () => any): void => {
+    canValidate.value = false
+    cb()
+    resetValidation()
```

Comment:
Why would a function that executes callback without validation resetValidation? This likely must be named withValidationRest().

This must look like this.
```suggestion
    canValidate.value = true
```

Reply:
The suggested change won't work, I've tested it.

Reply:
You likely need to do something like this

```ts
const reset = withoutValidation(() => {
   // do stuff here
   resetValidation()
})
```